src/components/panel/panel.scss

```diff
@@ -28,41 +28,43 @@
 .md-panel {
   opacity: 0;
   overflow: auto;
+  position: fixed;
 
   &._md-panel-shown {
     // Only used when custom animations are present.
     // Overridden by the default animations.
     opacity: 1;
     transition: none;
+  }
 
-    &._md-panel-animate-slide-in {
-      opacity: 1;
-      transition: $swift-ease-out, opacity 1ms;
-    }
-
-    &._md-panel-animate-scale-in {
-      opacity: 1;
-      transition: $swift-ease-out;
-    }
-
-    &._md-panel-animate-fade-in {
-      opacity: 1;
-      transition: all 0.4s linear;
-    }
+  &._md-panel-animate-enter {
+    opacity: 1;
+    transition: all 0.5s cubic-bezier(0.0, 0.0, 0.2, 1);
   }
 
-  &._md-panel-animate-slide-out {
+  &._md-panel-animate-leave {
     opacity: 1;
-    transition: $swift-ease-out, opacity 1ms;
+    transition: all 0.5s cubic-bezier(0.4, 0.0, 1, 1);
   }
 
-  &._md-panel-animate-scale-out {
+  &._md-panel-animate-scale-out,
+  &._md-panel-animate-fade-out {
     opacity: 0;
-    transition: $swift-ease-out;
   }
 
-  &._md-panel-animate-fade-out {
+  &.md-opaque-enter {
+    height: 100%;
+    opacity: .48;
```

Comment:
Only used here and this is the value used by all backdrops so not likely to change.